setup.py

```diff
@@ -1,12 +1,35 @@
+import os
+import sys
+
 from setuptools import setup, find_packages
+
+if 'bsd' in sys.platform:
+    examplesdir = "/usr/local/share/examples/zabbix-smartmontools"
+    sudoersdir = "/usr/local/etc/sudoers.d"
+    sudoersfile = "Unix/sudoers.d/zabbix"
+elif sys.platform.startswith("linux"):
+    examplesdir = "/usr/local/share/doc/zabbix-smartmontools/examples"
+    sudoersdir = "/etc/sudoers.d"
+    sudoersfile = "Unix/sudoers.d/zabbix"
+else:
+    raise NotImplementedError
```

Comment:
Should just proceed without failure.

Reply:
Isn't it better to fail early rather than fail later due to undefined variable errors?  Perhaps you could fill in the proper values for Windows?

Reply:
```
elif sys.platform == 'win32':
    examplesdir = r"C:\zabbix-agent\zabbix-smartmontools"
    sudoersfile = "none"
```

Reply:
`error: can't copy 'none': doesn't exist or not a regular file`
Damn. It's too fragile - user have to correctly name the directory for it to work. Should just pass on Windows, but following code requires the vars. Can you work it out?

Reply:
Ok, try now.